src/cdomain/value/cdomains/mval.ml

```diff
@@ -36,12 +36,13 @@ struct
   let to_cil_exp lv = Lval (to_cil lv)
 
   let is_definite (_, o) = Offs.is_definite o
-  let top_indices (x, o) = (x, Offs.top_indices o)
 end
 
 module MakeLattice (Offs: Offset.Lattice): Lattice with type idx = Offs.idx =
 struct
   include MakePrintable (Offs)
+  
```

Comment:
Why was this moved from Printable?

Reply:
Because `Printable` indices in offsets aren't required to have a `top`, so `top_indices` cannot be defined for them. That's the case for the new `Z` offsets which are just `Printable`.
Everything else we had before (`IntDomain`, `Unit`, etc) just happened to have `top`.